gradle/osgi.gradle

```diff
@@ -16,7 +16,7 @@ afterEvaluate {
                     "org.mockito.*;version=${version}"
 
             instruction 'Import-Package',
-                    'net.bytebuddy.*;version=[1.4.26,2.0)',
+                    'net.bytebuddy.*;version="[1.4.26,2.0)"',
```

Comment:
@mgrafl 
this can be reduced to [1.4.26) which has an inclusive lower limit and an exclusive upper limit, denoting version 1.4.26 and any version after this, up to, but not including, version 2.0.0.